provider/cmd/pulumi-resource-aws-native/schema.json

```diff
@@ -5273,12 +5273,12 @@
                 "s3": {
                     "$ref": "#/types/aws-native:appflow:FlowS3SourceProperties"
                 },
-                "sAPOData": {
-                    "$ref": "#/types/aws-native:appflow:FlowSAPODataSourceProperties"
-                },
                 "salesforce": {
                     "$ref": "#/types/aws-native:appflow:FlowSalesforceSourceProperties"
                 },
+                "sapoData": {
+                    "$ref": "#/types/aws-native:appflow:FlowSAPODataSourceProperties"
```

Comment:
Should this change apply to type names as well? `SapoData` vs `FlowSAPODataSourceProperties` looks inconsistent?

Reply:
hmmm... yes, I think it should. I don't think we transform the type names at all today, but for consistency, we should.

Reply:
Since this review is already pretty big and is sufficient to resolve the linked issues, I'd like to ship this and then get the type names in a follow up. I filed https://github.com/pulumi/pulumi-aws-native/issues/1008 to track lowercasing initialisms in type names